specification/servicebus/resource-manager/Microsoft.ServiceBus/stable/2024-01-01/namespace-preview.json

```diff
@@ -0,0 +1,1171 @@
+{
```

Comment:
"/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ServiceBus/namespaces/{namespaceName}/failover is removed. 

Is this going to be added ? 

If users need to use this API, do they have to use old api versions ?

Reply:
Hey @ramoka178, This api present in preview version. we dont want to put it into stable version of api
So we removed it